code/modules/reagents/chemistry/reagents/toxin_reagents.dm

```diff
@@ -290,6 +290,7 @@
 	toxpwr = 0
 	taste_description = "sourness"
 	ph = 11
+	impure_chem = /datum/reagent/impurity/rosenol
```

Comment:
Just to note - how this is set up when the purity of this reagent is below 25% it will convert into Toxic monomers on consumption. If you don't want that set `inverse_chem = null`. I don't mind if you do, however! I'm just pointing it out in case.

Reply:
Changed, good that you mentioned it